Code/Mantid/Framework/API/inc/MantidAPI/IEventWorkspace.h

```diff
@@ -37,6 +37,7 @@ namespace API {
 */
 class MANTID_API_DLL IEventWorkspace : public MatrixWorkspace {
 public:
+  IEventWorkspace() {}
```

Comment:
Previously there was no constructor whatsoever, so a default constructor was generated by the compiler. Now there is a protected copy constructor, so we don't get the default constructor automatically.
You are of course right about the correction, I will add it.